src/api/assertion.js

```diff
@@ -78,13 +77,13 @@ export default class Assertion {
         return this._enqueueAssertion('lte', { expected, message, opts });
     }
 
-    within (start, finish, message, opts) {
+    within (expected, expected2, message, opts) {
```

Comment:
It's fine to keep `start` and `finish` here, because it has context. But in generic assertion command structure they will be converted to `expected` and `expected2`